### DISCUSSION TITLE
Start SEB - ERR_BLOCKED_BY_RESPONSE

### DISCUSSION THREAD

**Discussion by ArtemKislicyn** [SafeExamBrowser/seb-win-refactoring#304] [Q&A]
Hi,

when I try to start SEB and the exam page inside i see error: ERR_BLOCKED_BY_RESPONSE

What i can solve it?

![photo_2021-12-22_12-25-54](https://user-images.githubusercontent.com/66517153/147070572-08d7ac67-1179-40fa-add2-46c553571c2c.jpg)

**Comment by dbuechel** [comment]
Could you please provide the browser and client log files of the affected session? They can be found under `%LocalAppData%\SafeExamBrowser\Logs`.

**Reply by ArtemKislicyn** [reply]
Thanks, I found out the problem. This was refused to be displayed in an iframe because the "X-frame-Options" is set to "sameorigin"
